### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by ZacUSNYR** [jenkinsci/plasticscm-plugin#21]
We use Plastic cloud and have been unable to update the Jenkins plugin starting with 3 and the latest 3.1 build.

My selector setup
```
repository "dotnet@CompanyCloudAddress@cloud"
  path "/"
    smartbranch "/Development"
```

My build output 

```
[EnvInject] - Loading node environment variables.
Building in workspace C:\dir\project
[Project] $ "c:\Program Files\PlasticSCM5\client\cm" lwk --format={wkname}#{path}#{wkid}
......... removed workspace listing which was accurate.
[Project] $ "c:\Program Files\PlasticSCM5\client\cm" unco --all "C:\dir\project"
C:\dir\project\Namespace\Web\Common\VersionInfo.cs unchecked out correctly
C:\dir\project\Namespace\Web\Common unchecked out correctly
C:\dir\project\Namespace\Web\Project\Properties\AssemblyInfo.cs unchecked out correctly
C:\dir\project\Namespace\Web\Project\Properties unchecked out correctly
C:\dir\project\Namespace\Web\Project unchecked out correctly
C:\dir\project\Namespace\Web unchecked out correctly
C:\dir\project\Namespace unchecked out correctly
C:\dir\project unchecked out correctly
[Project] $ "c:\Program Files\PlasticSCM5\client\cm" ss "wk:Call Log-STAGE"
Selector for workspace Call Log-STAGE:
repository "dotnet@CompanyCloudAddress@cloud"
  path "/"
    smartbranch "/Development"

[Project] $ "c:\Program Files\PlasticSCM5\client\cm" update "C:\dir\project"
Downloading block of 7 files (42.75 KB) from dotnet@CompanyCloudAddress@cloud
... retrieves changes fine
[Project] $ "c:\Program Files\PlasticSCM5\client\cm" status --cset "C:\dir\project"
cs:3833@rep:dotnet@repserver:CompanyCloudAddress@cloud
FATAL: null
java.lang.NullPointerException
	at com.codicesoftware.plugins.hudson.PlasticSCM.checkout(PlasticSCM.java:206)
	at hudson.scm.SCM.checkout(SCM.java:504)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1208)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:574)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:499)
	at hudson.model.Run.execute(Run.java:1818)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
Started calculate disk usage of build
Finished Calculation of disk usage of build in 0 seconds
Started calculate disk usage of workspace
Finished Calculation of disk usage of workspace in  15 second
Finished: FAILURE
```

**Comment by mig42** [comment]
Hi, did you update your client to version 8.0.16.3400 or higher?

**Comment by massivetim** [comment]
Same issue here after upgrading to 3.1. We are using Plastic 8.0.16.3417.

**Comment by mig42** [comment]
We're reproducing this bug and we're fixing it right now. We'll release a fix ASAP!

**Comment by mig42** [comment]
Hi! This bug was fixed in our last release 3.2. Please upgrade your plugins and let us know if the issue is gone!

**Comment by ZacUSNYR** [comment]
> Hi! This bug was fixed in our last release 3.2. Please upgrade your plugins and let us know if the issue is gone!

Consider it resolved for me.  Thank you for the quick response.
